compiler/luci-interpreter/src/kernels/While.cpp

```diff
@@ -44,6 +44,27 @@ void While::configure()
   LUCI_INTERPRETER_CHECK(cond_outputs[0]->element_type() == DataType::BOOL);
 }
 
+void While::copy(const std::vector<const Tensor *> &src, const std::vector<Tensor *> &dst) const
```

Comment:
Optional:
These methods could be simple static functions, they do not use any fields/methods of While.
In that case there are no need to change `While.h`.

Reply:
I'll follow your suggestion :)